### DISCUSSION TITLE
Template section heading - User Stories | User Needs | User Experience ...

### DISCUSSION THREAD

**Discussion by shawna-slh** [w3c/low-vision-a11y-tf#140] [General]
Sub-discussion of: [Supplemental Guidance Template Discussion](https://github.com/w3c/low-vision-a11y-tf/discussions/137)

Context: [Draft Template](https://deploy-preview-21--wai-wcag-supplemental.netlify.app/lv/template/)

### Background

WAI initially used these short persona quotes in [What's New in WCAG 2.1](https://www.w3.org/WAI/standards-guidelines/wcag/new-in-21/#1411-non-text-contrast-aa). It includes a mini-persona, a quote for “problem” and a quote for “works well”.

There is no section heading, because that is the main content of the page.

#### In other contexts:
* User Need &mdash; COGA Supplemental Guidance uses “User Need”, as shown in this [in-progress COGA prototype](https://wai-wcag-supplemental.netlify.app/patterns/o1p03-consistent-design/)
* User Experience &mdash; For adding these persona quotes to the Understanding documents, EOWG suggested the heading: “User Experience”, as shown in this [since-revised Understanding prototype](https://w3c.github.io/wai-wcag-supporting-documents-redesign/2020-07-15-prototype-understanding-quote.html)

<hr>

For discussion: Is there a strong reason to use the same or different section heading? What are pros and cons?

**Comment by mraccess77** [comment]
Sam prefers "User Stories".
